resources/views/filament/components/server-console.blade.php

```diff
@@ -7,6 +7,20 @@
     <script src="https://cdn.jsdelivr.net/npm/@xterm/addon-search/lib/addon-search.min.js"></script>
     <script src="https://cdn.jsdelivr.net/npm/xterm-addon-search-bar/lib/xterm-addon-search-bar.min.js"></script>
     <link rel="stylesheet" href="{{ asset('/css/filament/server/console.css') }}">
+
+    @php
+        $userFont = auth()->user()->getCustomization()['console_font'] ?? 'ComicMono';
```

Comment:
Default is no longer `ComicMono`. Can it handle "no font"/ system default?
```suggestion
        $userFont = auth()->user()->getCustomization()['console_font'];
```